src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -1214,6 +1231,29 @@ public EntityDataModelDiff getEntityDataModelDiff( EntityDataModel edm ) {
         return Lists.newArrayList( result, conflicts );
     }
 
+    @Override
+    public Map<UUID, EntityTypePropertyMetadata> getAllEntityTypePropertyMetadata( UUID entityTypeId ) {
+        return entityTypePropertyMetadata
+                .entrySet( Predicates.equal( EntityTypePropertyMetadataMapstore.ENTITY_TYPE_INDEX, entityTypeId ) )
+                .stream()
+                .collect( Collectors.toMap( entry -> entry.getKey().getPropertyTypeId(), entry -> entry.getValue() ) );
```

Comment:
I generally prefer the `Entry::getValue` syntax over `entry -> entry.getValue()` syntax, but not a big deal!